runtime/exelib/common/libhlp.c

```diff
@@ -468,7 +468,7 @@ char * main_vmVersionString(void)
 
 	if ((minorVersion % 10) == 0)
 		minorVersion /= 10;
-	sprintf (versionStringPtr, "%d.%d%s,%s %s", 
+	snprintf(versionStringPtr, sizeof(versionString), "%d.%d%s,%s %s",
 		EsVersionMajor, minorVersion, EsExtraVersionString, native, EsBuildVersionString);
```

Comment:
While you're here, I suggest removing `native` and initializing `minorVersion` in its declaration:
```c
	U_32 minorVersion = EsVersionMinor;
	char *versionStringPtr = GLOBAL_DATA(versionString);

	if (0 == (minorVersion % 10)) {
		minorVersion /= 10;
	}
	snprintf(versionStringPtr, sizeof(versionString), "%d.%d%s, %s",
		EsVersionMajor, minorVersion, EsExtraVersionString, EsBuildVersionString);
```